include/openthread/border_routing.h

```diff
@@ -307,6 +319,26 @@ otError otBorderRoutingGetOmrPrefix(otInstance *aInstance, otIp6Prefix *aPrefix)
  */
 otError otBorderRoutingGetPdOmrPrefix(otInstance *aInstance, otBorderRoutingPrefixTableEntry *aPrefixInfo);
 
+/**
+ * Gets the data of platform generated RA message processed..
+ *
+ * Only mRequirePd, mNumPlatformRaReceived, mNumPlatformPioProcessed and mLastPlatformRaMsec fields are used in the
+ * returned info.
```

Comment:
This comment is a little weird: All the 4 fields are mentioned, why do we say "Only"? I'm not sure the intention of this sentence. I suggest simply removing this line.

Reply:
Will remove in next commit, thanks!